builtin/logical/pki/crl_util.go

```diff
@@ -38,8 +58,9 @@ type revocationInfo struct {
 //
 // The CRL builder also tracks the revocation configuration.
 type crlBuilder struct {
-	m            sync.Mutex
-	forceRebuild uint32
+	m                     sync.Mutex
+	forceRebuild          uint32
```

Comment:
This looks to me like a boolean?  Could we make it a boolean - preferably an atomic boolean?